examples/ansible/install.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 The KubeOne Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Install KubeOne cluster
+  hosts: localhost
+  connection: local
+  roles:
+    - kubeone_config
+    - kubeone_install
+  vars:
+    kubeone_name: testcluster  # Give the cluster a name.
+    kubeone_versions_kubernetes: 1.15  # Kubernetes version to install.
+    kubeone_apiEndpoint_host: 0.0.0.0  # Address of the kubernetes endpoint to create.
```

Comment:
> Address of the kubernetes endpoint to create.

I'm not really sure I like the comment because it doesn't explain well what is it about. This is the endpoint that is used to access the cluster. It should be a load balancer between all control plane nodes.

I think it would sound better if you say something along

```
The address used to access the Kubernetes control plane. Preferably a load balancer with all control plane nodes.
```

Reply:
Oh, misinterpreted the option